ziti/cmd/ziti/cmd/edge/login.go

```diff
@@ -92,15 +92,21 @@ func (o *loginOptions) Run() error {
 
 	var host string
 	if len(o.Args) == 0 {
-		if defaultId := config.Identities[id]; defaultId != nil && !o.IgnoreConfig {
-			host = defaultId.Url
-			o.Printf("Using controller url: %v from identity '%v' in config file: %v\n", host, id, configFile)
-		} else {
-			if host, err = term.Prompt("Enter controller host[:port] (default localhost:1280): "); err != nil {
+		defaultHost := "localhost:1280"
```

Comment:
Why would we ever want this as a default? Seems odd. I know it matches our example configs, but this is wierd.

Reply:
The more I think about this the more I don't like it. It is nice for dev, but what if someone using ziti doesn't use 1280 as their default and they use something else. What if some other software uses 1280 and it causes odd errors.

Reply:
If we're not promoting it a standard port then I'll remove it

Reply:
fixed